docs/content/docs/recipes/angular.md

```diff
@@ -1,6 +1,285 @@
+
+![Four browsers with the Timestamp app open in them one second after the button has been pushed.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-2.PNG)
+
+{{< callout note >}}
+
+This tutorial assumes that you are familiar with the [Fluid Framework Overview]({{< relref "/docs/_index.md" >}}) and that you have completed the [QuickStart]({{< relref "quick-start.md" >}}). You should also be familiar with the basics of [Angular](https://angular.io), [creating Angular projects](https://angular.io/guide/setup-local), and [Angular Hooks](https://angular.io/guide/lifecycle-hooks).
+
+{{< /callout >}}
+
+
+## Prerequisites
+
+1. Node.js must be installed on your local machine. To install, follow the instructions [here](https://nodejs.org/en/download/).
+1. Angular CLI must be installed on your local machine. To install, run the following command (after Node.js is installed).
+
+    ```dotnetcli
+    npm install -g @angular/cli
+    ```
+
+## Create the project
+
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project; e.g., `c:\My Fluid Projects`.
+1. Run the following command at the prompt.
+    ```dotnetcli
+    ng new fluid-angular-tutorial
+    ```
+
+1. The project is created in a subfolder named `fluid-angular-tutorial`. Navigate to it with the command `cd fluid-angular-tutorial`.
+1. The project uses two Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    |fluid&#x2011;framework&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;    |Contains the SharedMap [distributed data structure]({{< relref "dds.md" >}}) that synchronizes data across clients. *This object will hold the most recent timestamp update made by any client.*|
+    |fluidframework/tinylicious&#x2011;client&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   |Defines the connection to a Fluid service server and defines the starting schema for the [Fluid container][].|
+    &nbsp;
+
+    Run the following command to install the libraries.
+
+    ```dotnetcli
+    npm install @fluidframework/tinylicious-client fluid-framework
+    ```
+
+## Code the project
+
+1. Open the file `\src\app\app.component.ts` in your code editor. Delete all the default `import` statements. Then delete the line declaring the `title` property. The file should look like the following:
+
+    ```js
+    @Component({
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.css']
+    })
+    export class AppComponent {
+    }
+    ```
+
+1. Add the following `import` statements at the beginning of the file:
+
+    ```js
+    import { Component, OnInit, OnDestroy } from '@angular/core';
+    import { SharedMap } from "fluid-framework";
+    import { TinyliciousClient } from '@fluidframework/tinylicious-client';
+    ```
+
+1. Define the following `interface` under the import statements:
+    ```js
+    interface TimestampDataModel { time: string | undefined; }
```

Comment:
Code samples in docs are generally expected to be javascript (except some cases that are explicitly ts), so the typing information here should be removed.  The actual demo code in the repo is good as is though

Reply:
Should I also remove the type definitions for the section after this one?
```js
    sharedTimestamp: SharedMap | undefined;
    localTimestamp: TimestampDataModel | undefined;
    updateLocalTimestamp: (() => void) | undefined;
```
to 
```js
    sharedTimestamp;
    localTimestamp;
    updateLocalTimestamp;
```